src/graphql/node/status.rs

```diff
@@ -67,9 +68,14 @@ async fn load(
         let (resource_usage, ping) =
             fetch_resource_usage_and_ping(agent_manager, hostname, is_manager).await;
 
+        let encoded;
+        {
+            let key = node.unique_key();
+            encoded = OpaqueCursor(key.to_vec()).encode_cursor();
+        }
```

Comment:
Why would you just declare `encoded` in advance and initialize it later? Additionally, I don't see any advantage to limit the lifetime of `key` within the block.

Reply:
I'll change it.
Thank you.